types/object-map/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for object-map 1.0
+// Project: https://github.com/xixixao/object-map
+// Definitions by: Wolfgang Faust <https://github.com/wolfgang42>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function objectMap<I, O>(
```

Comment:
I suggest not to use `I`, `O`, it looks like `1`, `0`

Reply:
What do you suggest using instead? I picked them for `I`nput and `O`utput but I'm open to other suggestions.

Reply:
I believe `TInput` and `TOutput` is OK.